webpack.config.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
+const webpack = require("webpack");
+const { BundleAnalyzerPlugin } = require("webpack-bundle-analyzer");
+const { merge } = require("webpack-merge");
+
+const bnJsReplaces = [
+  "tiny-secp256k1",
+  "asn1.js",
+  "create-ecdh",
+  "miller-rabin",
+  "public-encrypt",
+  "elliptic",
+];
+
+function getDefaultConfiguration() {
+  return {
+    cache: true,
+    performance: {
+      hints: false,
+      maxEntrypointSize: 512000,
```

Comment:
In future commits I will be tweaking this value so that we have a way to catch future inflations of bundle size